preview-src/pullRequestOverviewRenderer.ts

```diff
@@ -563,6 +608,9 @@ class ReviewNode {
 		userLogin.href = this._review.user.url;
 		userLogin.textContent = this._review.user.login;
 
+		const authorAssociation: HTMLSpanElement = document.createElement('span');
+		authorAssociation.textContent = `(${this._review.authorAssociation.toLocaleLowerCase()})`;
```

Comment:
`authorAssociation` seems to be undefined when using the REST API. Also, when the authorAssociation is `none`, I think we shouldn't render it